### PR TITLE
feat: support groq translator

### PR DESCRIPTION
**Pull request by umm233**
Support Groq to translate

### CODE REVIEW

**Comment by yihong0618** [comment]
can you add some info to README? @umm233

File: book_maker/cli.py [hunk 1]
```diff
@@ -423,7 +433,7 @@ def main():
         if not options.api_base:
             raise ValueError("`api_base` must be provided when using `deployment_id`")
         e.translate_model.set_deployment_id(options.deployment_id)
-    if options.model == "openai":
+    if options.model in ("openai", "groq"):
```
Comment (yihong0618) [inline book_maker/cli.py:436]: this seems wrong it always raise error

**Comment by umm233** [comment]
ok, I also want to adjust the format of readme to make it clearer.

File: book_maker/cli.py (hunk 1 above)
Comment (umm233) [reply book_maker/cli.py:436]: Maybe you need run with --model_list arg.

Comment (umm233) [reply book_maker/cli.py:436]: Because Groq needs to specify the model, I reuse the way of specifying the model.

**Comment by yihong0618** [comment]
copy that merging this first

**Pull request merged**